plugins/available/alias-completion.plugin.bash

```diff
@@ -49,7 +49,7 @@ function alias_completion {
 		# (leveraging that eval errs out if $alias_args contains unquoted shell metacharacters)
 		eval "local alias_arg_words; alias_arg_words=($alias_args)" 2> /dev/null || continue
 		# avoid expanding wildcards
-		read -ra alias_arg_words <<< "$alias_args"
+		read -a alias_arg_words <<< "$alias_args"
```

Comment:
Do we need a `# shellcheck disable=SC2162` or does the previous one above cover this line too?

Reply:
This passes the lint stage, so I guess we do not need a new shellcheck directive